src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -209,59 +197,47 @@ interface NetworkAlertProps {
   thin?: boolean
 }
 
+const BETA_TAG_COLORS: { [chainId in SupportedL2ChainId]: string } = {
```

Comment:
I understand that `CHAIN_INFO` may get bloated but have you considered putting that there to colocate chain info and make the code easier to navigate for newer engs?